blog/2024-08-27-changed-icon-translations-schema.md

```diff
@@ -0,0 +1,33 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Changes to the icon translations schema"
+---
+
+The icon translations schema has been adjusted to allow assigning icons for sections in services.
```

Comment:
Could "sections in services" be a link to the docs describing what the sections are?

Reply:
Sure, added

Reply:
Great, thanks!